src/server/blockchain/blockchain-types.js

```diff
@@ -66,7 +66,7 @@ export interface Subscribe<T> {
         id: string;
         subscribe(callback?: Callback<Subscribe<T>>): Subscribe<T>;
         unsubscribe(callback?: Callback<boolean>): void | boolean;
-        arguments: object;
+        arguments: any;
```

Comment:
Maybe `{}` works better in these cases? 
https://flow.org/en/docs/types/objects/#toc-unsealed-objects